instrumentation/google.golang.org/grpc/otelgrpc/stats_handler.go

```diff
@@ -147,10 +147,19 @@ func (c *config) handleRPC(ctx context.Context, rs stats.RPCStats, isServer bool
 	}
 
 	switch rs := rs.(type) {
-	case *stats.Begin:
```

Comment:
Why is this removed?

Reply:
It is not doing anything (the statement is empty), so it's covered by `default`.